### PR TITLE
unwind currentTag

### PR DESCRIPTION
**Pull request by alexlamsl**
Also addresses a similar case with `minifyCSS`

### CODE REVIEW

File: tests/minifier.js [hunk 1]
```diff
@@ -987,6 +987,10 @@
   });
 
   test('script minification', function() {
+    input = '<script></script>foo';
```
Comment (kangax) [inline tests/minifier.js]: What was happening with this before the fix?

Comment (alexlamsl) [inline tests/minifier.js]: See #435 - it performs `minifyJS` on `foo` and gives `<script></script>foo;`

**Comment by alexlamsl** [comment]
improved test case for `minifyJS` so it's more clear what is being tested

**Comment by alexlamsl** [comment]
@kangax FYI the two tests were run against original `htmlminifier.js` and both failed

**Comment by kangax** [comment]
Hm, I'm still not sure about the point of this. Why are script contents outside of script elements should be minified?

**Comment by alexlamsl** [comment]
Without this PR, `minify('<style></style>this + is(not, code)', {minifyJS:true})` would alter those text, same with `minifyCSS`

So I am fixing it so that it does not do that anymore, hence the tests expecting the inputs to be untouched.

**Comment by alexlamsl** [comment]
IOW, contents outside of script elements should _not_ be minified, but they are currently being minified on `master`.

**Comment by kangax** [comment]
Oh wow, didn't know. Then it's a serious bug, of course, thanks!

**Pull request merged**